src/components/Header/Header.vue

```diff
@@ -0,0 +1,43 @@
+<template>
```

Comment:
I propose a simplification of this component to something like:

```vue
<template>
  <header class="Header">
    <nav class="NavBar">
      <a href="/" class="Logo">
        <img
          src="@/assets/img/logotip-anem-per-feina.png"
          alt="Logotip Anem per feina"
        />
      </a>
    </nav>
  </header>
</template>

<script>
export default {
  name: 'Button'
}
</script>

<style scoped lang="scss">
.Header {
  width: 100vw;
  background: #746ea9;
}

.NavBar {
  width: 100%;
  max-width: 1920px;
  margin: 0 auto;
  padding: 0.75em 1em;
}

.Logo {
  display: inline-block;
  margin-right: 1rem;
}
</style>
```
This way you there are fewer elements (you avoid the `.container`) that were there just for styling.

My recommendation is to avoid "extra" elements. In 99% of cases there is a way to achieve the same result by changing the CSS, but don`t get obsessed with this... there is a choice that you find that 1% and an extra "container" is necessary for styling reasons, just use them as the last resort.

In addition, paddings using percentages are not very desirable because those are [based in the parent element width](https://css-tricks.com/oh-hey-padding-percentage-is-based-on-the-parent-elements-width/).

Regarding the inner spacing of the navbar, I changed it to one following this spacing schema (inset stretch in this case):

![image](https://user-images.githubusercontent.com/13970905/96301989-49d60180-0ff8-11eb-9136-1c976e67f02b.png)
> source: [space in design systems](https://medium.com/eightshapes-llc/space-in-design-systems-188bcbae0d62) by Nathan Curtis (a very recommendable read)

All in all, these are just little improvements that I wanted to share with you, your code was perfectly valid, be proud of it!

Reply:
The image of the spacing schema is so interesting. I saved for my next component! Thx